v1/BetterRTX_Installer.sh

```diff
@@ -0,0 +1,433 @@
+Thanks For Installing BetterRTX! If you have any issues, use the #betterrtx-help forum channel in the Minecraft RTX Server!
+YOU STILL NEED AN RTX RESOURCE PACK FOR THIS TO WORK!
+
+END
+)
+
+# Uninstall prompts
+
+DownloadingVanilla="Downloading Latest Vanilla $STUB and $TONEMAP..."
+
+UninstallPrompt=$(cat <<-END
+Done :(
+We're Sorry to See You Go. If you have any suggestions or issues, create a message in the #betterrtx-help forum channel in the Minecraft RTX Server.
+
+END
+)
+
+Links=$(cat <<-END
+Invite Link: https://discord.gg/minecraft-rtx-691547840463241267
+Help Channel Link: https://discord.com/channels/691547840463241267/1101280299427561523
+
+END
+)
+
+function start {
+
+    if _checkForNotoFonts; then
+        if _checkForLolcat; then
+            echo "$Logo_Fancy" | lolcat --spread=1.0
+        else
+            echo "$Logo_Fancy"
+        fi
+        echo "$LogoExt_Fancy"
+    else
+        if _checkForLolcat; then
+            echo "$Logo" | lolcat --spread=1.0
+        else
+            echo "$Logo"
+        fi
+        echo "$LogoExt"
+    fi
+
+    echo "$InstallOptions"
+
+    while true; do
+         echo "$SelectPrompt"; read Selection
+        if _installMethod $Selection; then
+            break
+        else
+            continue
+        fi
+    done
+
+    if [ "$UNINSTALLBIT" = false ]; then
+        if _checkForAMD; then
+            echo $InstallNoteAMD
+        fi
+
+        if _checkForIntel; then
+            echo $InstallNoteIntel
+        fi
+    fi
+
+    echo $QuickNote
+    echo $QuickNote2
+
+    while true; do
+        echo $GetInstallLocation; read DefaultInstallLocation
+        if _checkInstallLocation $DefaultInstallLocation; then
+            break
+        else
+            continue
+        fi
+    done
+
+    _validateFiles
+    _moveFiles
+
+    if [ "$UNINSTALLBIT" = false ]; then
+        echo "$InstallPrompt"
+    else
+        echo "$UninstallPrompt"
+    fi
+
+    echo "$Links"
+}
+
+function cleanUp {
+    if [[ -s "$tempdir/$STUB" ]]; then
+        rm "$tempdir/$STUB"
+    fi
+
+    if [[ -s "$tempdir/$TONEMAP" ]]; then
+        rm "$tempdir/$TONEMAP"
+    fi
+}
+
+function _moveFiles {
+    echo $ReplacingStub
+    mv "$tempdir/$STUB" "$DefaultInstallLocation/data/renderer/materials/$STUB"
+
+    echo $ReplaceTonemap
+    mv "$tempdir/$TONEMAP" "$DefaultInstallLocation/data/renderer/materials/$TONEMAP"
+}
+
+function _validateFiles {
+    if [[ ! -s "$tempdir/$STUB" ]]; then
+        echo "$tempdir/$STUB $ValidationFailed"
+        exit 127
+    fi
+
+    if [[ ! -s "$tempdir/$TONEMAP" ]]; then
+        echo "$tempdir/$TONEMAP $ValidationFailed"
+        exit 127
+    fi
+}
+
+function _checkForNotoFonts {
+    NOTOFONTS_DIR="/usr/share/fonts/noto"
+
+    if [ -d "$NOTOFONTS_DIR" ]; then
+        return
+    else
+        false
+    fi
+}
+
+function _checkForLolcat {
+    if [ -f "/bin/lolcat" ]; then
+        return
+    else
+        false
+    fi
+}
+
+function _checkForBinary {
```

Comment:
Maybe use `pkg-config` to check for binaries? Some distributions are immutable and have multiple end points for their binaries.